core/src/test/java/org/testcontainers/containers/ExposedHostTest.java

```diff
@@ -100,4 +93,16 @@ protected void assertResponse(GenericContainer<?> container, int port) {
             container.stop();
         }
     }
+
+    private ContainerDef tinyImage() {
```

Comment:
```suggestion
    private ContainerDef tinyContainer() {
```

This now describes a container, not only an image, right? Or more correct even `tinyContainerDef` (since it is not an actual container, but only the definition).